src/components/CourseDetail/CourseDetail.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from "react-router-dom";
+import axios from 'axios';
+
+import ExperienceCard from '../ExperienceCard/ExperienceCard';
+
+/* CourseDetail Component to render the page when a course is selected from a
+    search */
+const CourseDetail = (props) => {
+
+    const location = useLocation();
+    const expObj = location.state.expObj;
+    const imgLink = location.state.imgLink;
+    const api_url = 'http://localhost:8080/es-api/more-like-this/';
+    let cardSection = (
+        <div>
+            Error Loading Related card.
+        </div>
+    )
+
+    // state to keep track of all the related course found
+    const [coursesState, setCoursesState] = useState({
+        coursesObj: null,
+        isLoading: false,
+        error: null
+    });
+
+    // Fetch similar courses from elastic search
+    useEffect(() => {
+        let url = api_url + expObj.meta.id;
+        // set the loading state
+        setCoursesState(previousState => {
+            const resultState = {
+                coursesObj: null,
+                isLoading: true,
+                error: null
+            }
+            return resultState
+        });
+        axios.get(url)
+            .then(response => {
+                console.log(response.data);
+                setCoursesState(previousState => {
+                    return {
+                        coursesObj: response.data,
+                        isLoading: false,
+                        error: null
+                    }
+                });
+            })
+            .catch(err => {
+                console.log(err)
+                setCoursesState(previousState => {
+                    return {
+                        coursesObj: null,
+                        isLoading: false,
+                        error: err
+                    }
+                })
+            });
+    }, [expObj.meta.id])
+
+    // showing loading text when the api call is in progress
+    if (coursesState.isLoading === true) {
+        cardSection = (
+            <div className="center-text">Loading...</div>
+        )
+    // once the api call is done and it's not an error we load the previews
+    } else if (coursesState.coursesObj && coursesState.isLoading === false) {
+        cardSection = (
+            coursesState.coursesObj.hits.map((course, idx) => {
+                return <ExperienceCard courseObj={course} key={idx} />
+            })
+        );
+    }
+
+    return (
+        <div className="content-section">
+            <div className="row content-panel course-detail">
+                <div className="inner-content">
+                    <h2>{expObj.Course.CourseTitle}</h2>
+                    <div className="row">
+                        <div className="col span-2-of-5">
+                            <img src={imgLink} alt="test"/>
```

Comment:
The `alt` attribute value will need to be something other than "test" -- Per 508, it should describe either the image, or if it pertains to a hyperlink it should describe the action to be taken.

Reply:
I was planning on handling these under the 508 review ticket but I can go ahead and fix this now